tests/puppeteer/campaigns/functional/BO/05_customerService/02_orderMessages/03_pagination.js

```diff
@@ -0,0 +1,140 @@
+require('module-alias/register');
+// Using chai
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+const OrderMessageFaker = require('@data/faker/orderMessage');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const OrderMessagesPage = require('@pages/BO/customerService/orderMessages');
+const AddOrderMessagePage = require('@pages/BO/customerService/orderMessages/add');
+// Test context imports
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_customerService_order_messages_pagination';
+
+let browser;
+let page;
+let numberOfOrderMessages = 0;
+const createOrderMessageData = new OrderMessageFaker();
+
+// Init objects needed
+const init = async function () {
+  return {
+    boBasePage: new BOBasePage(page),
+    loginPage: new LoginPage(page),
+    dashboardPage: new DashboardPage(page),
+    orderMessagesPage: new OrderMessagesPage(page),
+    addOrderMessagePage: new AddOrderMessagePage(page),
+  };
+};
+
+/*
+Create 11 order messages
+Paginate between pages
+Delete order messages with bulk actions
+ */
+describe('Order messages pagination', async () => {
+  // before and after functions
+  before(async function () {
+    browser = await helper.createBrowser();
+    page = await helper.newTab(browser);
+    this.pageObjects = await init();
+  });
+  after(async () => {
+    await helper.closeBrowser(browser);
+  });
+  // Login into BO
+  loginCommon.loginBO();
+
+  it('should go to order messages page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToOrderMessagesPage', baseContext);
+    await this.pageObjects.boBasePage.goToSubMenu(
+      this.pageObjects.boBasePage.customerServiceParentLink,
+      this.pageObjects.boBasePage.orderMessagesLink,
+    );
+    await this.pageObjects.boBasePage.closeSfToolBar();
+    const pageTitle = await this.pageObjects.orderMessagesPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.orderMessagesPage.pageTitle);
+  });
+
+  it('should reset all filters and get number of order messages in BO', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'resetFilterFirst', baseContext);
+    numberOfOrderMessages = await this.pageObjects.orderMessagesPage.resetAndGetNumberOfLines();
+    if (numberOfOrderMessages !== 0) await expect(numberOfOrderMessages).to.be.above(0);
+  });
+
+  /* eslint-disable no-loop-func */
```

Comment:
I seen that you disable it really often, maybe you could disable it in your eslint config if this is boring for you ? That would make sense 

ESLint configuration should be adapted to your need, if you consider that using loops is usefull for you.

Reply:
Or use a correct code : https://eslint.org/docs/rules/no-loop-func (see samples)

Reply:
To much vars out of scope (and linked to mocha too) to use the samples so I used a foreach